jobs/transforms/question_validation.py

```diff
@@ -21,27 +21,51 @@
 
 from core.domain import question_domain
 from core.platform import models
+from jobs import job_utils
 from jobs.decorators import validation_decorators
 from jobs.transforms import base_validation
 
 (question_models,) = models.Registry.import_models([models.NAMES.question])
 
 
 @validation_decorators.AuditsExisting(
-    question_models.QuestionSnapshotMetadataModel,
-    question_models.QuestionCommitLogEntryModel)
-class ValidateQuestionCommitCmdsSchema(
+    question_models.QuestionSnapshotMetadataModel)
+class ValidateQuestionSnapshotMetadataModel(
         base_validation.BaseValidateCommitCmdsSchema):
     """Overrides _get_change_domain_class for question models."""
 
     def _get_change_domain_class(self, input_model): # pylint: disable=unused-argument
-        """Returns a Change domain class.
+        """Returns a change domain class.
 
         Args:
             input_model: datastore_services.Model. Entity to validate.
 
         Returns:
-            change_domain.BaseChange. A domain object class for the
+            question_domain.QuestionChange. A domain object class for the
             changes made by commit commands of the model.
         """
         return question_domain.QuestionChange
+
+
+@validation_decorators.AuditsExisting(
+    question_models.QuestionCommitLogEntryModel)
+class ValidateQuestionCommitLogEntryModel(
+        base_validation.BaseValidateCommitCmdsSchema):
+    """Overrides _get_change_domain_class for question models."""
```

Comment:
Please update the docstring according to the class name(here and elsewhere).

Reply:
Done